internal/driver/nodeserver.go

```diff
@@ -316,5 +316,5 @@ func (ns *LinodeNodeServer) NodeGetInfo(ctx context.Context, req *csi.NodeGetInf
 }
 
 func (ns *LinodeNodeServer) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGetVolumeStatsRequest) (*csi.NodeGetVolumeStatsResponse, error) {
```

Comment:
probably dont need `ctx` in the definition here on line 318 either? (and anywhere this method is being called from)

Reply:
We can't remove that because NodeGetVolumeStats() is a pre define func from node plugin interface. We need to adhere to the upstream interfacce